### ISSUE TITLE
 Expected subtree parent to be a mounted class component. This error is likely caused by a bug in React. Please file an issue

### ISSUE THREAD

**Issue by ShallowGreen** [michalochman/react-pixi-fiber#71]
### Description
 Expected subtree parent to be a mounted class component. This error is likely caused by a bug in React. Please file an issue

### Steps to reproduce

1.
```
import React, { Component } from 'react';
import { Stage } from "react-pixi-fiber";

class TOScada extends Component {
    constructor(props){
        super()
        this.state = {
            width: 0,
            height: 0
        }
    }

    render() {
        return (
            <div>
                <Stage width={800} height={600} options={{ backgroundColor: 0x10bb99 }}>
                </Stage>
            </div>
        );
    }
}

export default TOScada;

```
2.Expected subtree parent to be a mounted class component. This error is likely caused by a bug in React. Please file an issue

### Additional info

- react-pixi-fiber version: 0.4.9
- React version:16.5.0
- ReactDOM version:16.5.0
- PixiJS version:4.8.2

**Comment by michalochman** [comment]
@ShallowGreen is there a particular reason you render empty `Stage`?

**Comment by michalochman** [comment]
Ok, I can see that it actually breaks with anything on React 16.5, I'll have a look

**Comment by ShallowGreen** [comment]
https://codesandbox.io/s/q7oj1p0jo6
the demo can not work as well

**Comment by ShallowGreen** [comment]
👍 Thanks for your reply :)

**Comment by michalochman** [comment]
Seems like it's not working after this change: https://github.com/facebook/react/pull/13397

I need to update `react-reconciler` to the newest version to fix this.

**Comment by faizaand** [comment]
Downgrading to React 16.4.2, from 16.5.2, solved the problem for me.

**Comment by michalochman** [comment]
@ShallowGreen this is now fixed for React 16.5  in `react-pixi-fiber@0.5.0`

**Comment by kasperkronborg** [comment]
Looks like this has been reintroduced in 0.6.2 and with React 16.5.2. At least I was getting the error. Downgraded to react-pixi-fiber@0.5.1 solved it.
